pkg/sink/sink_test.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 
 	"github.com/gorilla/mux"
-	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	pipelinev1alpha1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
```

Comment:
Why did you want to keep this one as v1alpha1 instead of upgrading to v1beta1?

Reply:
Because the usage here is for the `PipelineResource` which is staying in v1alpha1 and not moving to v1beta1

Reply:
Ahh, right 😄